rskj-core/src/main/java/co/rsk/peg/BridgeSerializationUtils.java

```diff
@@ -293,19 +297,29 @@ public static ABICallElection deserializeElection(byte[] data, AddressBasedAutho
         return new ABICallElection(authorizer, votes);
     }
 
-    public static byte[] serializeLockWhitelist(LockWhitelist whitelist) {
-        List<Address> whitelistAddresses = whitelist.getAddresses();
-        int serializationSize = whitelistAddresses.size() * 2 + 1;
+    public static byte[] serializeOneOffLockWhitelist(LockWhitelist whitelist) {
```

Comment:
I'm thinking names here are a bit confusing. If I were to use this, I'd think this would sort of serialize the whole whitelist rather than filtering and then serializing. I would maybe consider passing in the list of entries to serialize and then do the filtering on the storage side.